packages/core/src/Stryker.ts

```diff
@@ -101,14 +104,45 @@ export default class Stryker {
       } finally {
         // `injector.dispose` calls `dispose` on all created instances
         // Namely the `SandboxPool`, `MutantTranspileScheduler` and `ChildProcessProxy` instances
-        await statisticsProcess.sendStatistics();
         await mutationTestProcessInjector.dispose();
         await LogConfigurator.shutdown();
       }
     }
     return Promise.resolve([]);
   }
 
+  private async collectStatistics(mutantResults: MutantResult[]) {
```

Comment:
Ik mis nog uitgebreide testen voor de statistieken. Er zit vrij veel logica in die mij wel belangrijk lijkt om ook goed te testen

Reply:
https://github.com/Jelle28/stryker/pull/1/commits/854478c6332d0e7d608535e301b710617c7c6afa